src/lib/crypto/common/index.ts

```diff
@@ -9,3 +9,10 @@ export async function exportKey(
     const exp = await webcrypto.subtle.exportKey(format, key)
     return utils.arrBufToBase64(exp)
 }
+
+export async function exportKeyBytes(
+    key: CryptoKey, 
+    format: ExportKeyFormat
```

Comment:
We don't need to allow ExportKeyFormat to be configurable, we're going to be very opinionated on our key formats. Our exports and imports should always be the nicely PEM formatted version of PKCS#8.

Reply:
This specific method is supposed to handle both asymm and symm keys, so format needs to be configurable (e.g. webcrypto doesn't support exporting symm keys to Pkcs8).

Reply:
I can make use more opnionated everywhere else we call this function though

Reply:
This function has not changed -- calls to this function are opinionated based on context (using pub vs priv vs symm keys)